cdc/generation.hh

```diff
@@ -150,9 +150,8 @@ db_clock::time_point new_generation_timestamp(bool add_delay, std::chrono::milli
 // using `mutation_size_threshold` to decide on the mutation sizes. The partition key of each mutation
 // is given by `gen_uuid`. The timestamp of each cell in each mutation is given by `mutation_timestamp`.
 //
-// Works for only specific schemas: CDC_GENERATIONS_V2 (in system_distributed_keyspace)
-// and CDC_GENERATIONS_V3 (in system_keyspace).
-future<utils::chunked_vector<mutation>> get_cdc_generation_mutations(
+// Works only for the CDC_GENERATIONS_V2 schema (in system_distributed_keyspace).
```

Comment:
Let's take this opportunity to fix the typo (`system_distributed_keyspace` -> `system_distributed keyspace`)